### DISCUSSION TITLE
Change logo, CSS and translation

### DISCUSSION THREAD

**Discussion by Martin511** [CorentinTh/enclosed#145] [Ideas]
Hello.
Thank you for this!!
I really have not much skills so my question may be stupid: Is it possible to change / edit the logo, colors and translate the text to my own language?

**Comment by CorentinTh** [comment]
Hi @Martin511 
The translation system landed just few minutes ago in #146, you can eaily contribute to the i18n in https://github.com/CorentinTh/enclosed/tree/main/packages/app-client/src/locales, current languages are English and French

Regarding the logo and colors, you can fork the project and update the content, but it's not intended to be nativelly configurable

**Reply by Martin511** [reply]
Hi.

As I have no push access, I have attached a German translation below
[de.json](https://github.com/user-attachments/files/17070139/de.json)

If you want, you can add it.
I´m not familiar with Github and I don´t know how to fork an develop things.

But I just want to contribute a little bit ;-)

It´s a nice simple tool. Thank you!


Regards
Martin

**Comment by Mohamedhazeem** [comment]
Your question isn't stupid at all—it's a great one! Yes, it is definitely possible to change the logo, colors, and translate text on most websites. 
**To Change the Logo->**
Locate the <img> tag or CSS background-image property in your HTML or CSS files that points to the current logo then replace the src (or URL) with the path to your new logo image.
`<img src="path-to-your-new-logo.png" alt="Logo">`
**To Edit Colors->**
Open your CSS file and find the classes or elements responsible for setting the colors (like background, text, or button colors).
Update the color or background-color property to your desired color.
```
body {
    background-color: #f0f0f0;
    color: #333;
}
```
**To Translate Text->**
Simply locate the text in your HTML files that you want to translate and replace it with the text in your language.
For dynamic content, it might involve using a localization plugin or library, depending on the framework you're using (like _i18n_ for JavaScript).
if you're looking for a simpler, more visual way to edit elements like colors or logos, you could try [TweakSync](https://tweaksync.dev/). It lets you style your webpage directly from Chrome in real-time and sync those changes back to your code in VS Code. No advanced coding skills required, just visual tweaks. You can check it out on the [Chrome Web Store](https://chromewebstore.google.com/detail/tweaksync/ponbgdgplifbgjianglonaondpkonnio?authuser=0&hl=en-GB) and [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=TweakSync.tweaksync).
